.github/workflows/test.yml

```diff
@@ -0,0 +1,19 @@
+
+name: test
+on:
+- pull_request
+jobs:
+  vapor_macos:
+    runs-on: macos-latest
+    env:
+      DEVELOPER_DIR: /Applications/Xcode_11.4.1.app/Contents/Developer
+    steps:
```

Comment:
```suggestion
    steps:
    - name: Select latest available Xcode
      uses: maxim-lobanov/setup-xcode@1.0
      with:
        xcode-version: latest```

Reply:
@siemensikkema  The tests are failing with this change.

Reply:
@emarashliev yes, the trailing ``` are the problem

Reply:
Didn't notice it 🤦‍♂️😀